tagmanager/ctags/parse.h

```diff
@@ -128,7 +128,8 @@ extern void checkRegex (void);
 
 /* Extra stuff for Tag Manager */
 extern tagEntryFunction TagEntryFunction;
-extern void setTagEntryFunction(tagEntryFunction entry_function);
+extern void *TagEntryUserData;
```

Comment:
exposing this is consistent with the function itself, but not used and "leaking" a few more ctags' guts.  So if we could rather remove the `TagEntryFunction` global export than adding the data one, it'd be awesome :)